src/FondOfSpryker/Yves/CheckoutPage/Form/Steps/PaymentForm.php

```diff
@@ -39,7 +39,7 @@ protected function addPaymentMethodChoices(FormBuilderInterface $builder, array
     /**
      * @return \Spryker\Yves\StepEngine\Dependency\Plugin\Form\SubFormPluginCollection|\Spryker\Yves\StepEngine\Dependency\Form\SubFormInterface[]
      */
-    protected function getPaymentMethodSubForms(): array
+    protected function getPaymentMethodSubForms()
```

Comment:
The doc block suggests that this function returns an array, are you sure that this should be removed as type hint?

Reply:
I removed the typehint because otherwise phpstan reports the following error 

"PHPDoc tag @return with type array<Spryker\Yves\StepEngine\Dependency\Form\SubFormInterface>|Spryker\Yves\StepEngine\Dependency\Plugin\Form\SubFormPluginCollection is not subtype of native type array. "

Reply:
Remove the collection part in the docblock and just use `\Spryker\Yves\StepEngine\Dependency\Form\SubFormInterface[]` instead.